docs/en/latest/plugins/openid-connect.md

```diff
@@ -2,10 +2,10 @@
 title: openid-connect
 keywords:
   - APISIX
-  - Plugin
+  - API Gateway
   - OpenID Connect
-  - openid-connect
-description: This document contains information about the Apache APISIX openid-connect Plugin.
+  - OIDC
+description: OpenID Connect (OIDC) is an authentication protocol based on the OAuth 2.0. It allows the client to obtain user information from the identity provider (IdP), e.g., Keycloak, Ory, Okta, Auth0, etc. API Gateway APISIX supports to integrate with the above identity providers to protect your APIs.
```

Comment:
This description should ideally be around 160 characters. I guess that is best in terms of SEO. The license checker fails because the description is too big.